packages/ui/src/common/components/selects/select.tsx

```diff
@@ -43,7 +43,6 @@ export const Select = <T extends any>({
   useEffect(() => {
     if (isDefined(selected)) {
       setSelectedOption(selected)
-      onChange(selected)
```

Comment:
I removed this so the `onChange` handler applies the new filter only once per change and not on the first render. I checked with a few select through the app, and it doesn't appear to impact their behavior.